tfhe/src/integer/server_key/radix_parallel/tests_unsigned.rs

```diff
@@ -2135,7 +2135,7 @@ where
     // hard-coded tests
     // 10, 7, 14 are from the paper and should trigger different branches
     // 16 is a power of two and should trigger the corresponding branch
-    let hard_coded_divisors: [u32; 4] = [10, 7, 14, 16];
+    let hard_coded_divisors: [u64; 4] = [10, 7, 14, 16];
```

Comment:
can you comment on this change ?

Reply:
We encrrypt 32 bits integers, however for parameters 3_3 we actually encrypt more than 32 bits, which will trigger the new assert so we have to use a u64 scalar

Reply:
I'm fine with the change anyways as I saw we were casting to u64